controllers/dspipeline_params.go

```diff
@@ -265,6 +277,7 @@ func (p *DSPAParams) SetupObjectParams(ctx context.Context, dsp *dspa.DataScienc
 		)
 		p.ObjectStorageConnection.Port = config.MinioPort
 		p.ObjectStorageConnection.Scheme = config.MinioScheme
+
```

Comment:
current implementation will crash when using default minio because `.secure` is never set, and will default to `nil` since we use a pointer now, to avoid this, we need to manually set this value to false for this case: 

```go
p.ObjectStorageConnection.Secure = BoolPointer(false)  // we can assume it will be false since we are deploying it
```